### DISCUSSION TITLE
Obtain full destination URL

### DISCUSSION THREAD

**Discussion by noh626** [dotnet/yarp#2294] [Q&A] [answered]
Hi,

I've noticed that the ASP.NET Core ILogger logs the full destination URL when proxying requests, as shown below:

`info: Yarp.ReverseProxy.Forwarder.HttpForwarder Proxing to https://xxx
`

Getting the destination URL prefix is not an issue, but I'm looking for a way to obtain the full destination URL. Is there a method to access this complete URL within a middleware component, or perhaps through an interface like IForwarderTelemetryConsumer?

**Comment by Tratcher** [comment, marked as answer]
See https://github.com/microsoft/reverse-proxy/blob/2077576b271064c7d14f306c4ed345e4c5d02f24/src/ReverseProxy/Forwarder/HttpForwarder.cs#L438

**Reply by noh626** [reply]
Thanks 👍
